src/js/graphql/gql/areaById.ts

```diff
@@ -47,6 +47,20 @@ export const QUERY_AREA_BY_ID = gql`
       }
       pathTokens  
       ancestors
+      organizations{
+        orgId
```

Comment:
Then we would need to retrieve `orgType` here.

Reply:
Also remember to add a space after `organizations` and `content` before the curly braces. I guess our linter doesn't check graphql strings. 

An organization fragment might be nice at some point, but e can break that out later on when we need to reuse it.

Reply:
true!

Reply:
is `organizations` field available for climb query as well?